lightcycle-integration-test/src/main/java/com/soundcloud/lightcycle/sample/basic/AppCompatActivityLogger.java

```diff
@@ -0,0 +1,63 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityCallback;
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityLifecycleCallback;
+
+class AppCompatActivityLogger extends BaseActivityLogger<AppCompatActivity> {
```

Comment:
Oh wait, sorry ! That's why it compiles, it is a LightCyckle. 
I thought `AppCompatActivityLogger` was an `Activity`. 

Scratch this : )